lib/ddtrace/contrib/httparty/integration.rb

```diff
@@ -0,0 +1,35 @@
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/httparty/configuration/settings'
+require 'ddtrace/contrib/httparty/patcher'
+
+module Datadog
+  module Contrib
+    module HTTParty
+      # Description of HTTParty integration
+      class Integration
+        include Contrib::Integration
+        register_as :httparty
+
+        def self.version
+          Gem.loaded_specs['httparty'] && Gem.loaded_specs['httparty'].version
+        end
+
+        def self.present?
+          super && defined?(::HTTParty)
+        end
+
+        def self.compatible?
+          super && Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.0.0')
```

Comment:
I don't think we need to explicitly check the Ruby version here.

This check is only necessary if the user is able to install the gem (`httparty`) with a specific version of Ruby that would impact the integration.

In the case of `httparty`, I don't see any impediments that would hinder tracing, as long as the gem is successfully installed.